Mockery v2 build tags with vendors

Hi,

We are using `mockery` to generate mocks for multiple packages in [aws-controllers-k8s/Makefile](https://github.com/aws/aws-controllers-k8s/blob/main/Makefile#L61-L62).  One of the dependencies we have needs a specific build tag `codegen` [aws-sdk-go/private/model/api](https://github.com/aws/aws-sdk-go/blob/master/private/model/api/api.go#L1) . 
Mockery `v1.x.x` looks to be perfectly working with the `-tag=codegen` however with using mockery `v2.x.x` with the flag `--tags=codegen` doesn't seem to be working the same way.

Logs trying to generate mocks for `pkg/generate/config`:
```
mockery --tags="codegen" --case=underscore --output=mocks/pkg/generate --dir=./pkg/generate/config --all
24 Sep 20 12:04 CEST INF Starting mockery dry-run=false version=2.2.1
24 Sep 20 12:04 CEST INF Walking dry-run=false version=2.2.1
24 Sep 20 12:04 CEST ERR Error parsing file error="/Users/ahilaly/source/github.com/aws/aws-controllers-k8s/pkg/generate/config/config.go:210:60: Operation not declared by package api" dry-run=false version=2.2.1
```
I couldn't find an issue similar to this one. Is there any solution or work around for this issue? 

Thank you in advance!

Related issues: aws/aws-controllers-k8s#265

--- a/pkg/config/config.go
+++ b/pkg/config/config.go
@@ -5,7 +5,7 @@ var SemVer = "0.0.0-dev"
 
 type Config struct {
 	All           bool
-	BuildTags     string
+	BuildTags     string `mapstructure:"tags"`
 	Case          string
 	Config        string
 	Cpuprofile    string

